src/hooks/screen/useCreateChat.ts

```diff
@@ -124,6 +124,22 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      console.log({e}, 'error create chat');
+    } finally {
+      setLoadingCreateChat(false);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function has a catch block that logs the error to the console but does not handle it in any other way. This could lead to silent failures that are hard to debug or understand from a user's perspective. Consider throwing the error after logging it, or handling it in a way that is visible to the user.

```diff
  } catch (e) {
-   console.log({e}, 'error create chat');
+   console.log({e}, 'error create chat');
+   throw e;
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->